metron-interface/metron-alerts/scripts/run-all-test.sh

```diff
@@ -14,10 +15,19 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 #
-port: 4200
+current_path=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )
+cd "$current_path/.."
 
-rest:
-  host: localhost
-  port: 4200
+source ~/.nvm/nvm.sh; nvm install; node --version
 
-dirPath: dist
\ No newline at end of file
+npm ci
+
+npm run test:ci
+npm run cypress:ci
+
+# licence check needs an initial build step
+npm run build
+echo "┌─────────────────────────────────────┐"
+echo "| Checking Apache license headers...  |"
+echo "└─────────────────────────────────────┘"
+./scripts/prepend_license_header.sh
```

Comment:
This change for some reason identified as a rename/move. It is a removal of yml config file no longer needed and the addition of run-all-test.sh.